dockstore-webservice/src/main/java/io/dockstore/webservice/DockstoreWebserviceApplication.java

```diff
@@ -369,6 +392,36 @@ public void run(DockstoreWebserviceConfiguration configuration, Environment envi
 
     }
 
+    private void describeAvailableLanguagePlugins(DefaultPluginManager languagePluginManager) {
+        List<PluginWrapper> plugins = languagePluginManager.getStartedPlugins();
+        if (plugins.isEmpty()) {
+            LOG.info("No language plugins installed");
+            return;
+        }
+        StringBuilder builder = new StringBuilder();
+        builder.append("PluginId\tPlugin Version\tPlugin Path\tInitial Path Pattern\tPlugin Type(s)\n");
```

Comment:
A small nit. For readability this section could use `String.format` in addition to the append so that we can avoid lines that are just `builder.append("\t")`.